inc/class-s3-uploads.php

```diff
@@ -142,6 +142,17 @@ public function s3() {
 			$params['region'] = $this->region;
 		}
 
+		if ( defined('WP_PROXY_HOST') && defined('WP_PROXY_PORT') ) {
+			$proxy_auth = '';
+			$proxy_address = WP_PROXY_HOST . ':' . WP_PROXY_PORT;
+
+			if ( defined('WP_PROXY_USERNAME') && defined('WP_PROXY_PASSWORD') ) {
+				$proxy_auth = WP_PROXY_USERNAME . ':' . WP_PROXY_USERNAME . '@';
```

Comment:
Shouldn't this be
    
    $proxy_auth = WP_PROXY_USERNAME . ':' . WP_PROXY_PASSWORD . '@';